pkgs/os-specific/darwin/dockutil/default.nix

```diff
@@ -1,29 +1,45 @@
-{ lib, stdenv, fetchurl, libarchive, p7zip }:
-stdenv.mkDerivation rec {
+{ lib
+, stdenv
+, fetchFromGitHub
+, swift
+, swiftpm
+, swiftpm2nix
+, darwin
+}:
+
+let
+  generated = swiftpm2nix.helpers ./generated;
```

Comment:
This should be turned into a proper wrapper and should only generate one file.

Reply:
Can you provide more context and details around your thinking on this, @SuperSandro? Is it something that is already being worked on, where contributions would be helpful and appreciated or is it something that needs to be thought out and began from scratch?

Reply:
I don't know, you probably should do some research in the open PRs to see if you can find something already